validator/app_versions.json

```diff
@@ -135,7 +138,10 @@
          "45.0a1", "45.0a2", "45.0", "45.*",
          "46.0a1", "46.0a2", "46.0", "46.*",
          "47.0a1", "47.0a2", "47.0", "47.*",
-         "48.0a1", "48.0a2", "48.0", "48.*"
+         "48.0a1", "48.0a2", "48.0", "48.*",
+         "49.0a1", "49.0a2", "49.0", "49.*",
+         "50.0a1", "50.0a2", "50.0", "50.*",
+         "51.0a1", "51.0a2", "51.0", "51.*"
```

Comment:
I don't think there will be any release versions of Thunderbird until 52. Also, I'm fairly certain we can remove support for all applications except Firefox Desktop since we haven't had compat tests for any of these in years.